### PR TITLE
Apply user defined values to form

### PR DESCRIPTION
**Pull request by flashbackzoo**
Applies user set values (config and custom validation) to the front-end form

### CODE REVIEW

File: templates/UserForm.ss [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <% include UserFormProgress %>
 
-<form class="userform" $AttributesHTML>
+<form
+	$AttributesHTML
+	class="userform"
+	data-livevalidation='$EnableLiveValidation'
```
Comment (tractorcow) [inline templates/UserForm.ss]: Shall we override getAttributes() on UserForm and set these there?

Comment (flashbackzoo) [inline templates/UserForm.ss]: yes - we shall..

File: code/forms/UserForm.php [hunk 2]
```diff
@@ -170,4 +170,20 @@ public function validate() {
...
+	/**
+	 * Override some we can add UserForm specific attributes to the form.
+	 *
+	 * @return array
+	 */
+	public function getAttributes() {
+		$attrs = parent::getAttributes();
+
+		$attrs['class'] = $attrs['class'] . ' userform';
+		$attrs['data-livevalidation'] = $this->controller->EnableLiveValidation === '1';
```
Comment (tractorcow) [inline code/forms/UserForm.php]: `$attrs['data-livevalidation'] = (bool)$this->controller->EnableLiveValidation`;

Comment (assertchris) [inline code/forms/UserForm.php]: `$attrs['data-livevalidation'] = filter_var($this->controller->EnableLiveValidation, FILTER_VALIDATE_BOOLEAN)`

Comment (tractorcow) [inline code/forms/UserForm.php]: Or that too. :P

**Pull request merged**